flang/lib/Lower/OpenMP/Utils.cpp

```diff
@@ -24,6 +24,12 @@ llvm::cl::opt<bool> treatIndexAsSection(
     llvm::cl::desc("In the OpenMP data clauses treat `a(N)` as `a(N:N)`."),
     llvm::cl::init(true));
 
+llvm::cl::opt<bool> enableDelayedPrivatization(
+    "openmp-enable-delayed-privatization",
+    llvm::cl::desc(
+        "Emit `[first]private` variables as clauses on the MLIR ops."),
+    llvm::cl::init(false));
```

Comment:
Yes, it is temporary until we get delayed privatization working end-to-end. Since delayed privatization is quite disruptive, it has to be introduced carefully and be tested extensively enough first.